### PR TITLE
[RNMobile] remove clear all settings button from Image block

### PR DESCRIPTION
**Pull request by jbinda**
## Description

This PR removes `Clear all settings` button from Image block BottomSheet settings according to [this issue](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2243)

Please also refer to:
[Related gutenberg-mobile PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/2250)
[Related gutenberg-mobile issue](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2243)

## How has this been tested?

1. Use default `initial-html`
2 Expect BottomSheet settings in Image block do not have `Clear all settings` action on the bottom

## Screenshots 
before | after
-- | --
 | 

## Types of changes

Refactor

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

**Review by etoledom — APPROVED** [review]
Looks great! 🎉 

Thank you @jbinda 
Let's wait for a resolution on https://github.com/wordpress-mobile/gutenberg-mobile/issues/2243 before merging 👍

**Review by dratwas — APPROVED** [review]

**Comment by pinarol** [comment]
hey @dratwas 👋  Could you merge this one if there's not a special reason that prevents us?

**Pull request merged**
